integration-test/test/handler-commands.bats

```diff
@@ -21,6 +21,16 @@ teardown(){
     [[ "$diff" = *"A /var/lib/azure/custom-script"* ]]
```

Comment:
@boumenot ok this patch basically keeps the path same but deletes it first for images who are captured with remnants of the old handler. guarantee here is "install is executed only once".

what you're suggesting is essentially changing the path altogether. the way I see we execute it is:  in the "enable" command we have a migration code, such as:

    mv -rf /var/lib/azure/custom-script /var/lib/waagent/custom-script

When the hotfix landed to existing VMs, it will move the state to `/var/lib/waagent/custom-script` next time it got enabled (VM reboot or goal state update etc). Then we'd start using this path... For clean VMs, the migration would be a noop.

Do you wanna do this?